src/homework13/main.java

```diff
@@ -0,0 +1,28 @@
+package homework13;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+
+public class main {
+    public static void main(String[] args) {
+        boolean result = validatorXMLSchema("D:/ONLINE_freeIT/src/homework13/devices.xsd", "D:/ONLINE_freeIT/src/homework13/devicesCatalogue.xml");
```

Comment:
Lifehack
you can use relative path to files so it will be system independent and path independent
relative path example "./freeIT/src/homework13/devices.xsd"
File.pathSeparator - helps you to do path system independent
One more option - put files in resource dir:
URL url = getClass().getResource("ListStopWords.txt");
File file = new File(url.getPath());

Reply:
easier then it seems. helped changing
boolean result = validatorXMLSchema("./src/homework13/devices.xsd", "./src/homework13/devicesCatalogue.xml");

thanks!!!